### DISCUSSION TITLE
Issue Adding Bond to Homebridge

### DISCUSSION THREAD

**Discussion by JayRuhMe** [aarons22/homebridge-bond#151] [General]
I stopped using my Bond for a while, but after updating it today with the latest code I'm seeing a weird error when I try start the service. Turned on debug and this is what it shows:

Error (185e458b244e8000) [get http://192.168.87.30/v2/sys/version] - {"message":"timeout of 1ms exceeded","name":"Error","stack":"Error: timeout of 1ms exceeded\n    at createError (/usr/local/lib/node_modules/homebridge-bond/node_modules/axios/lib/core/createError.js:16:15)\n    at RedirectableRequest.handleRequestTimeout (/usr/local/lib/node_modules/homebridge-bond/node_modules/axios/lib/adapters/http.js:280:16)\n    at RedirectableRequest.emit (events.js:315:20)\n    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge-bond/node_modules/follow-redirects/index.js:170:12)\n    at listOnTimeout (internal/timers.js:554:17)\n    at processTimers (internal/timers.js:497:7)","config":{"url":"http://192.168.87.30/v2/sys/version","method":"get","data":"{}","headers":{"Accept":"application/json, text/plain, */*","BOND-Token":"REDACTED","Bond-UUID":"REDACTED","Content-Type":"application/json;charset=utf-8","User-Agent":"axios/0.21.1","Content-Length":2},"transformRequest":[null],"transformResponse":[null],"timeout":1,"xsrfCookieName":"XSRF-TOKEN","xsrfHeaderName":"X-XSRF-TOKEN","maxContentLength":-1,"maxBodyLength":-1,"axios-retry":{"retryCount":3,"lastRequestTime":1619574700369}},"code":"ECONNABORTED"}

I did change the network it's connected to now, but I've verified that I can ping the address no problem. I was also able to http to the IP address of the device. Last thing... I noticed HTTP above references a "V2" directory (http://192.168.87.30/v2/sys/version). I'm not able to get to that directory either, BUT I am able to get to the same link replacing "V2" with "V1".  It seems to take me to a Bond registration page of sorts. 

Would appreciate any help you might be able to give.

**Comment by JayRuhMe** [comment]
Oh, and this error follows the one from the original post:

0 cached accessories were loaded
(node:4362) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'bondid' of undefined
    at BondPlatform.getDevices (/usr/local/lib/node_modules/homebridge-bond/dist/platform.js:47:65)
    at /usr/local/lib/node_modules/homebridge-bond/dist/platform.js:39:26
    at Array.forEach (<anonymous>)
    at /usr/local/lib/node_modules/homebridge-bond/dist/platform.js:38:28
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:4362) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 11)
(node:4362) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by richieboyle** [comment]
I have this same issue. Did you get a resolution? Happened after a restart of my EERO mesh network.

**Reply by richieboyle** [reply]
Let me know, as I was using it on a daily basis and the issue only begun after a power outage and restart.

**Comment by JayRuhMe** [comment]
Turned out my device was coming up on one of my guest networks after a reset. Got it on the right network and everything worked as it should. Good luck.
